.github/workflows/semantic-release.yaml

```diff
@@ -0,0 +1,25 @@
+name: Semantic Release
+
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:    
+      - 'main'
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    concurrency: release
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Python Semantic Release
+      uses: relekang/python-semantic-release@master
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        repository_username: __token__
+        repository_password: ${{ secrets.PYPI_TOKEN }}
```

Comment:
Will this work? Did you add any secretes in the current repo?

Reply:
Hi @fgypas , it works now, but not completely as planned. The release note that it generated does not contain meaningful message for the new version. I may leave it like this and see what happens to next release.